lib/jekyll/netlify/environment.rb

```diff
@@ -0,0 +1,21 @@
+module Jekyll
+  module Netlify
+    class Environment
+      def initialize
+        if Jekyll.env.empty?
```

Comment:
need to check for evidence of Netlify first.

If `DEPLOY_URL` or something else says it is Netlify, the default is production.